src/capybara.js

```diff
@@ -249,12 +249,18 @@ Capybara = {
   },
 
   isNodeVisible: function(node) {
-    while (node) {
-      var style = node.ownerDocument.defaultView.getComputedStyle(node, null);
-      if (style.getPropertyValue('display') == 'none' || style.getPropertyValue('visibility') == 'hidden')
-        return false;
+    var style = node.ownerDocument.defaultView.getComputedStyle(node, null);
+    // Only check computed visibility style on current node since it
```

Comment:
Well that's confusing!

Reply:
@jferris Yep - and thinking about it, it probably screws with visible text too, not that's worth attempting to fix (or possible)